extensions/objects/templates/BooleanEditor.js

```diff
@@ -31,7 +31,7 @@ oppia.directive('booleanEditor', ['$compile', function($compile) {
     },
     link: function(scope, element) {
       scope.getTemplateUrl = function() {
-        return OBJECT_EDITOR_TEMPLATES_URL + 'Boolean';
+        return '/object_editor_template/Boolean';
```

Comment:
Could you instead make this an Oppia constant (like you did with RULE_SUMMARY_WRAP_CHARACTER_COUNT above)? I don't think it's a good idea to have so many sources of truth for this URL, especially since we were discussing consolidating the URL definitions between the frontend and the backend and not duplicating this URL in-line will help with the process.